frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -655,6 +661,18 @@ public boolean isSnapshotDisabled() {
         return snapshotDisabled;
     }
 
+    public boolean isSSLEnabled() {
+        String inferenceAddress = prop.getProperty(TS_INFERENCE_ADDRESS, "http://127.0.0.1:8080");
```

Comment:
@dhaniram-kshirsagar Why is the url hardcoded here? Can we remove it?

Reply:
This is the default value (i.e. http) for the cases when the user has not supplied any address via config or env properties.

Reply:
Can there be a mix of ssl and non ssl endpoints e.g. `http://127.0.0.1:8080` and `https://127.0.0.1:8444`? Either we should clarify or handle that case

Reply:
@maaquib can you please elaborate on this.

Reply:
@dhaniram-kshirsagar We now have 3 endpoints for TS -- Management API, Inference API, Metrics API. If user selects combination of http/https across the three will that be supported? It will be better to have all endpoints of same type